core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -192,6 +194,35 @@ class CityInfo {
         return 0
     }
 
+    fun isGrowing(): Boolean {
+        return cityStats.currentCityStats.food > 0 && cityConstructions.currentConstruction != Constants.settler
+    }
+
+    fun isStarving(): Boolean {
+        return cityStats.currentCityStats.food < 0
+    }
+
+    /** Take -1 to mean infinity. */
+    fun getNumTurnsToNewPopulation(): Int {
```

Comment:
Maybe instead of `-1` you can return `null` in case the city is not growing? Might be more expressive, not sure though

Reply:
I did try to return `null`, but Android Studio was throwing errors about returning null in a non-null type, so I went with `-1` instead.

Reply:
Yeah, if you return `null` the type returned by the functions should be `Int?` to reflect the nullability of the returned type.